README.md

```diff
@@ -114,12 +114,15 @@ Available block directives include:
 - `@public` marks a function public. In most plugins, functions are private by
   default, though this default may be overridden on a per-plugin basis.
 - `@private` marks a function private.
-- `@section name[, id]` allows you to write a new section for the helpfile. The
-  id will be a lowercased version of name if omitted.
+- `@section name[, id]` allows you to write a new section for the
+  helpfile. The id will be a lowercased version of name if omitted.
+- `@parentsection id` defines the current section as a child of the given
+  section. Must be contained within a @section block.
```

Comment:
Add backticks around this `@section` for code font, and the other `@parentsection` below. GitHub seems to be highlighting them strangely and code font is a little clearer anyway.